Converters/Autocad/Speckle.Converters.AutocadShared/ToSpeckle/Geometry/HatchToSpeckleConverter.cs

```diff
@@ -78,25 +78,19 @@ public SOG.Region Convert(ADB.Hatch target)
 
   private ADB.Curve PolylineFromLoop(ADB.HatchLoop loop)
```

Comment:
Can you add a comment at the top of the class, indicating that `Hatch` are complex classes with many edge cases, and the most robust conversion method is converting them to a `Region` first.

Reply:
Similarly, can you explain that the `Region.CreateFromCurves()` method on line 33 expects database-resident objects, and therefore requires conversion of hatch loops into database entities.